metadata.rb

```diff
@@ -6,12 +6,12 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '4.2.0'
 depends          'java'
-depends          'apache2'
+depends          'apache2', '~> 5.2.1'
```

Comment:
This should probably just be < 6 so we don't constrain things too much. We just can't rely on 6.x